src/shared/db.ts

```diff
@@ -642,4 +644,14 @@ export class DB {
 
         return entry
     }
+
+    public async fetchEventDefinitions(): Promise<EventDefinitionType[]> {
+        return (await this.postgresQuery('SELECT * FROM posthog_eventdefinition', undefined, 'fetchAllPluginLogs'))
```

Comment:
I guess the tag here lingered from a paste?

Reply:
whoops, fixed